test/functional/apps/management/_index_pattern_results_sort.js

```diff
@@ -25,89 +25,70 @@ export default function ({ getService, getPageObjects }) {
   const PageObjects = getPageObjects(['settings', 'common']);
 
   describe('index result field sort', function describeIndexTests() {
-    before(function () {
+    before(async function () {
       // delete .kibana index and then wait for Kibana to re-create it
-      return kibanaServer.uiSettings.replace({});
+      await kibanaServer.uiSettings.replace({});
     });
 
     const columns = [{
       heading: 'Name',
       first: '@message',
       last: 'xss.raw',
-      selector: function () {
-        return PageObjects.settings.getTableRow(0, 0).getVisibleText();
+      selector: async function () {
+        return await PageObjects.settings.getTableRow(0, 0).getVisibleText();
       }
     }, {
       heading: 'Type',
       first: '_source',
       last: 'string',
-      selector: function () {
-        return PageObjects.settings.getTableRow(0, 1).getVisibleText();
+      selector: async function () {
+        return await PageObjects.settings.getTableRow(0, 1).getVisibleText();
```

Comment:
nit: Just to note, we're not really gaining anything from converting this to an async function. Using async/await is only really beneficial when there is more promise handling in the function.